recipes/r-censusgeography/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = '0.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-censusgeography
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/censusGeography_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/censusGeography/censusGeography_{{ version }}.tar.gz
+  sha256: 2d5044073f36bbf32f1cb934b31644a02ef17a2f97bce0f778e1145a0b688fd9
+
+build:
+  ## dependencies not available on win
+  skip: True  # [win]
```

Comment:
Please don't skip windows if this is just noarch and the win dependencies are not available.

Change back to 'noarch: generic' (also update the other section + add bld.bat back). We can then ignore that win CI failure and still merge it. The real build + upload in the feedstock runs on Linux and makes the package as it is noarch available everywhere, where the dependencies are available.

Reply:
Done, thank you!